src/com/axis/rtspclient/RTSPClient.as

```diff
@@ -311,7 +311,9 @@ package com.axis.rtspclient {
 
         if (state === STATE_SETUP) {
           /* this is not the case when falling through, e.g. SETUP of first track */
-          if (!/RTP\/AVP\/TCP;unicast;interleaved=/.test(parsed.headers["transport"])) {
+          if (!(/^RTP\/AVP\/TCP;/.test(parsed.headers["transport"]) &&
+        	/unicast;/.test(parsed.headers["transport"]) &&
+        	/interleaved=/.test(parsed.headers["transport"]) )){
```

Comment:
The semi-colon should be part of the regex test. They merely indicate separation of parameters. If e.g. RTP/AVP/TCP is the last parameter, it may not end in a semi-colon.